### PR TITLE
Fix: Price List - Issue when imagePosition is undefined when used as a reusable block.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://trello.com/c/a1ETE3OZ/196-price-list-issue-when-used-as-a-reusable-block

### Types of changes

- Bug fix

### How has this been tested?

- Create a price list as a reusable block in master ad images.
- Drag and drop on diff pages, you will see the issue.
- Switch to this branch to refresh the page where the price list reusable block is used. Issue resolved

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/price-list-child/edit.js [hunk 1]
```diff
@@ -129,10 +129,18 @@ class UAGBRestaurantMenuChild extends Component {
...
-		
-		const parentClientId = select( 'core/block-editor' ).getBlockHierarchyRootClientId( this.props.clientId );
+
+		const parentClientId = select( 'core/block-editor' ).getBlockHierarchyRootClientId( this.props.clientId );	
 		const parentAttributes = select('core/block-editor').getBlockAttributes( parentClientId );
-		
+
+		var Position = '';
+
+		if( parentAttributes ) {
```
Comment (RajkiranBagal) [inline src/blocks/price-list-child/edit.js:138]: Update the code with this.

let position = ( parentAttributes ) ? parentAttributes.imagePosition : imagePosition;

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: src/blocks/price-list-child/edit.js (hunk 1 above)
Comment (AbhijitNage123) [reply src/blocks/price-list-child/edit.js]: Resolved

**Review by RajkiranBagal — APPROVED** [review]

File: readme.txt [hunk 2]
```diff
@@ -166,8 +166,9 @@ The awesome React FontIconPicker is developed by [Alessandro Benoit](http://code
 
 == Changelog ==
 
-= 1.21.0 = 
+### 1.21.0 ###
```
Comment (vrundakansara) [inline readme.txt:169]: = 1.21.0 =

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: readme.txt (hunk 2 above)
Comment (AbhijitNage123) [reply readme.txt]: Updated

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**